packages/builder/src/pages/builder/portal/apps/[appId]/index.svelte

```diff
@@ -28,6 +34,20 @@
   }
 
   $: fetchScreens(app?.devId)
+
+  const receiveMessage = async message => {
+    if (message.data.type === "docLoaded") {
```

Comment:
I mention why we need a document load handler to begin with in the main body of the PR.


Putting an `on:load` handler on the iframe doesn't actually fire when the document loads, it seems to fire once it thinks "enough stuff is loaded to paint" which seemed to be once most scripts were downloaded. So I added this handler, and a script that fires the event to the client app doc.